src/c_vec.rs

```diff
@@ -273,12 +228,19 @@ impl<T> CSlice<T> {
     pub fn is_empty(&self) -> bool { self.len() == 0 }
 }
 
-impl<T> AsSlice<T> for CSlice<T> {
-    /// View the stored data as a slice.
-    fn as_slice<'a>(&'a self) -> &'a [T] {
```

Comment:
Why delete as_slice, your own project is depending on it.
https://github.com/GuillaumeGomez/rust-GSL/blob/master/src/types/combination.rs#L113